dig.php

```diff
@@ -4,242 +4,43 @@
     die('This command can only be used in CLI mode.');
 }
 
-function startDevelopmentServer($pidfile)
-{
-    $cmd = 'php -S localhost:8082';
-    $outputfile = '/dev/null';
-    echo 'location: ' . getcwd() . "\n";
-    shell_exec(sprintf("%s > %s 2>&1 & echo $! >> %s", $cmd, $outputfile, $pidfile));
-    sleep(1);
-}
-
-function killDevelopmentServer($pidfile)
-{
-    if (file_exists($pidfile)) {
-        $pids = file($pidfile);
-        foreach ($pids as $pid) {
-            shell_exec('kill -9 ' . $pid);
-        }
-        if (unlink($pidfile)) {
-            echo "Successfully deleted " . $pidfile;
-        } else {
-            echo "Problem deleting " . $pidfile;
-        }
-    }
-}
-
 date_default_timezone_set('Europe/Paris');
 mb_internal_encoding('UTF-8');
 
-// Manage CTRL+C and other script interruptions
-declare(ticks = 1); // how often to check for signals
-// This function will process sent signals
-function sig_handler($signo)
-{
-    global $git_mozorg;
-    global $svn_misc;
-    if ($signo == SIGTERM || $signo == SIGHUP || $signo == SIGINT) {
-        if ($signo != 15) {
-            print "Received signal $signo and will exit now!\n";
-        } else {
-            print "Operations finished\n";
-        }
-        // Cleanup
-        chdir($svn_misc);
-        exec('svn cleanup');
-        exec('pkill php');
-        exit;
-    }
-}
-
-// These define the signal handling
-pcntl_signal(SIGTERM, 'sig_handler');
-pcntl_signal(SIGHUP,  'sig_handler');
-pcntl_signal(SIGINT,  'sig_handler');
-
-// Repositories to loop for date in
-$app             = realpath(__DIR__);
-$pid_file        = $app . '/pidfile.txt';
-$data_path       = $app   . '/logs/data.json';
-$repos           = $app   . '/repos';
-$git_mozorg      = $repos . '/mozillaorg/';
-$svn_misc        = $repos . '/l10n-misc';
-$git_langchecker = $repos . '/langchecker';
-$git_stores      = $repos . '/appstores';
+// Declare paths and time
+$app               = realpath(__DIR__);
+$archive_data_path = $app . '/archive.json';
```

Comment:
OK, makes sense. I'd still prefer it outside of the root though (e.g. `/archive/data.json`)